### ISSUE TITLE
package list output broken?

### ISSUE THREAD

**Issue by matthiaskrgr** [nabijaczleweli/cargo-update#87]
cargo update used to print a list of installed crates
````
    Updating registry `https://github.com/rust-lang/crates.io-index`
Package                          Installed  Latest    Needs update
afl                              v0.4.0     v0.4.0    No
base100                          v0.4.1     v0.4.1    No
bat                              v0.4.1     v0.4.1    No
cargo-arch                       v0.1.0     v0.1.0    No
cargo-asm                        v0.1.16    v0.1.16   No
....
````
but since the latest update this no longer seems to be the case, is this intended?

````
Package  Installed  Latest  Needs update
Package  Installed  Latest  Needs update
````

**Comment by mati865** [comment]
Latest version introduced bug and packages without toolchain set are filtered away: 
```
$ cargo install-update -ag                                                                  
    Updating registry `https://github.com/rust-lang/crates.io-index`
                                                                                
Package  Installed  Latest  Needs update

No packages need updating.
Package  Installed  Latest  Needs update

No git packages need updating.


$ cargo install-update-config clippy -t nightly
Toolchain         nightly
Default features  true


$ cargo install-update -ag                       
    Updating registry `https://github.com/rust-lang/crates.io-index`

Package  Installed  Latest  Needs update

No packages need updating.
Package  Installed                                 Latest                                    Needs update
clippy   30a9879c461ce18ff120e513a2f6a16d0f2f4b80  30a9879c461ce18ff120e513a2f6a16d0f2f4b80  No

No git packages need updating.
```

**Comment by nabijaczleweli** [comment]
Fuck sake, I tested filters thoroughly but I didn't test it without filters.
Anyway, check `master`?

**Comment by mati865** [comment]
Works fine, thanks!

**Comment by matthiaskrgr** [comment]
Looks good to me, thanks for the quick fix! :D

**Comment by nabijaczleweli** [comment]
Oof, released in [`v1.6.1`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v1.6.1).
